client/src/featureform/resources.py

```diff
@@ -853,14 +1012,34 @@ def _create_feature_resource(self, db) -> None:
     def get_status(self):
         return ResourceStatus(self.status)
 
-    def is_ready(self):
-        return self.status == ResourceStatus.READY.value
+    def wait(self, timeout=None):
+        self.wait_function(self.name, self.variant, "feature", timeout)
```

Comment:
Why do we need to pass these args?

Reply:
for the function to call the metadata on the resource status.

Reply:
See comment above about passing the obj